framework/doc/content/syntax/FVKernels/index.md

```diff
@@ -0,0 +1,206 @@
+# FVKernels System
+
+For the finite volume method (FVM), `FVKernels` are the base class for `FVFluxKernel`, `FVElementalKernel`. These specialized objects satisfy the following tasks:
+
+* `FVFluxKernel` represents numerical fluxes evaluate on the element faces.
```

Comment:
This is minor, but I'm reading through all of the documentation and our list syntax is `-`.

Should be a quick find and replace. Also: the documentation is fantastic.